### ISSUE TITLE
Unable to easily copy IDs of my devices anymore

### ISSUE THREAD

**Issue by anoadragon453** [vector-im/element-web#20577]
### Steps to reproduce

https://user-images.githubusercontent.com/1342360/149631716-f6ab80b5-a7e1-4a30-a336-738c7e0e6f87.mp4

With the new device list design, it is no longer easy/possible to copy the ID of a given device. The device ID appears in a tooltip when hovering over device, but copying from a tooltip is not possible. Thus, one has to hover and display the tooltip, then manually type out the device ID in a separate window. This can be frustrating for devs or support staff that may need this information regularly.

Steps to reproduce:

1. Go to Settings -> Security & Privacy.
2. View list of devices.
3. Attempt to copy the ID of a device.

#### What did you expect?

A UI flow to do so without manually copying it.

#### What happened instead?

Manually copying was necessary.

There are many ways to enable this functionality. Some suggestions:

1. Display the device ID next to the device name in a different style (font size, colour). Allow copying as text.
    - Bonus points for simply clicking the ID allowing for a copy action.
    - This could also be gated behind the "developer mode" labs flag.
2. Much less discoverable, but holding down a modifier key (i.e shift) would display the ID, allowing copying the ID when click on the device name.

Similarly, it would be nice to extend the solution to the Security section of the profile sidebar, which also only displays the device ID in a tooltip.

### Operating system

Arch Linux

### Application version

Element Nightly version: 2022011101, Olm version: 3.2.8

### How did you install the app?

https://aur.archlinux.org/packages/element-desktop-nightly-bin/

### Will you send logs?

No

**Comment by anoadragon453** [comment]
May I suggest something like this?

![image](https://user-images.githubusercontent.com/1342360/153191348-d61fa759-2e3a-43ed-a7df-ffcd411b1075.png)

Perhaps it would only appear if "Developer mode" were enabled.

**Comment by anoadragon453** [comment]
Note that this is no longer an issue when using the new session manager. Session details can be expanded and the device ID is available as copy-pastable text.

**Comment by t3chguy** [comment]
Closing as fixed in labs
